tools/depends/target/openssl/Makefile

```diff
@@ -3,23 +3,23 @@ DEPS= ../../Makefile.include Makefile
 
 # lib name, version
 LIBNAME=openssl
-VERSION=1.0.2l
+VERSION=1.1.0h
 SOURCE=$(LIBNAME)-$(VERSION)
 ARCHIVE=$(SOURCE).tar.gz
 
 # configuration settings
-CONFIGURE=MACHINE=$(PLATFORM) ./config shared zlib --openssldir=$(PREFIX) --with-zlib-include=$(PREFIX)/include --with-zlib-lib=$(PREFIX)/lib
+CONFIGURE=MACHINE=$(PLATFORM) ./config no-shared zlib no-asm --prefix=$(PREFIX) --with-zlib-include=$(PREFIX)/include --with-zlib-lib=$(PREFIX)/lib
```

Comment:
imho we shouldn't disable asm where it works. historically it was disabled on ios and osx only.

Reply:
reason why I disabled it is the following build error

```/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_encrypt'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_decrypt'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_ecb_encrypt'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_ccm64_encrypt_blocks'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_ccm64_decrypt_blocks'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_ctr32_encrypt_blocks'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_xts_encrypt'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_xts_decrypt'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_cbc_encrypt'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_set_decrypt_key'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here
/usr/bin/ld.gold: error: ../tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libcrypto.a(aesni-x86_64.o): multiple definition of 'aesni_set_encrypt_key'
/usr/bin/ld.gold: /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu-debug/lib/libgnutls.a(aesni-x86_64.o): previous definition here```

Reply:
meh..
ok, I guess it's not worth it to try to patch around it.